packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsForm.tsx

```diff
@@ -114,7 +106,7 @@ export const ItemDetailsForm: React.FC<ItemDetailsFormProps> = ({
             pattern: { value: /^[0-9]+$/, message: t('error.invalid-value') },
             onChange: event => onChangeQuantity(Number(event.target.value)),
           })}
-          defaultValue={invoiceLine?.numberOfPacks}
+          defaultValue={summaryItem?.numberOfPacks || 0}
```

Comment:
Less of a regression and more of a TODO!

This is the quantity which needs to be changes for editing an invoice item, so not the invoiceLine.numberOPacks, anyway.